fuseimg/datasets/tests/test_datasets.py

```diff
@@ -107,9 +123,11 @@ def tearDown(self) -> None:
 
         shutil.rmtree(self.stoic21_cache_dir)
         
+        shutil.rmtree(self.isic_cache_dir)
+        shutil.rmtree(self.isic_data_dir)
```

Comment:
remove

Reply:
I changed it to match kits21:
`        if "ISIC19_DATA_PATH" not in os.environ:
            shutil.rmtree(self.isic_data_dir)
`